src/views/dialogs/SharePNLAnalyticsDialog.tsx

```diff
@@ -61,7 +63,12 @@ export const SharePNLAnalyticsDialog = ({
       await copyBlobToClipboard(blob);
 
       triggerTwitterIntent({
-        text: `Check out my ${assetId} position on @dYdX\n\n#dYdX #${assetId}\n[paste image and delete this!]`,
+        text: `${stringGetter({
+          key: STRING_KEYS.TWEET_MARKET_POSITION,
+          params: {
+            MARKET: assetId,
+          },
+        })}\n\n#dYdX #${assetId}\n[${stringGetter({ key: STRING_KEYS.TWEET_PASTE_IMAGE_AND_DELETE_THIS })}]`,
```

Comment:
hmmm... since the variable is same in both strings... at some point in the future should we just combine theres into one?

Reply:
ya ideally it'd be one string key, but these two already exist 😬 so didn't feel like making a new one haha